test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -2911,114 +2891,3 @@ rules:
         resources: ["configmaps"]
 `
 )
-
-func TestValidatingAdmissionPolicyTypeChecking(t *testing.T) {
```

Comment:
Add the integration test somewhere for policy typechecking?

Reply:
The problem with running the controller outside API server is that the integration test with API server becomes impossible. The controller was manually set up in the unit tests. Besides that, we will have to rely on the E2E test suite.

Reply:
Let's add it in https://github.com/kubernetes/kubernetes/pull/114514 or as a followup after #114514 merged :)